rpc/rpcserver/server.go

```diff
@@ -277,14 +279,42 @@ func (s *walletServer) ImportPrivateKey(ctx context.Context, req *pb.ImportPriva
 			"Only the imported account accepts private key imports")
 	}
 
-	_, err = s.wallet.ImportPrivateKey(wif, nil, req.Rescan, 0)
+	_, err = s.wallet.ImportPrivateKey(wif, nil, req.Rescan, req.ScanFrom)
```

Comment:
Before this, I would check for non-zero scan_from and error when rescan is false.